networking/configuring-a-custom-pki.adoc

```diff
@@ -0,0 +1,57 @@
+[id="configuring-a-custom-pki"]
+= Configuring a custom PKI
+include::modules/common-attributes.adoc[]
+:context: configuring-a-custom-pki
+
+toc::[]
+
+Some platform components (i.e., web console) use Routes for communication and,
```

Comment:
Avoid i.e. Suggest something like "Some platform components, such as the web console, use..."

Reply:
@bergerhoffer good to know. Does this also apply to `e.g.`, or is that one allowed?

Reply:
Yep, also should avoid e.g. - IBM style guide says to use "for example" instead.